src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1765,7 +1765,7 @@ private static bool AccessingAutoPropertyFromConstructor(BoundExpression receive
             var propertyIsStatic = propertySymbol.IsStatic;
 
             return (object)sourceProperty != null &&
-                    sourceProperty.IsAutoProperty &&
+                    sourceProperty.IsAutoPropertyOrUsesFieldKeyword &&
```

Comment:
I'm not so sure that this is 100% correct. There's a usage in `AbstractFlowPass` that visits the backing field when this returns true; are we sure we want that behavior for all flow passes?

Reply:
Good catch, thanks. For now, I've limited this change to the one caller that expects this method to include `field` cases.